Scripts/Modules/SelectionModule/SelectionModule.cs

```diff
@@ -135,10 +135,12 @@ GameObject GetGroupRoot(GameObject hoveredObject)
             if (m_GroupMap.TryGetValue(hoveredObject, out groupParent))
                 return groupParent;
 
+#if UNITY_EDITOR
```

Comment:
We could use our instantiation helper to insert a dummy component that can act as a group root in runtime mode